src/components/Picker/BasePicker.tsx

```diff
@@ -0,0 +1,210 @@
+         *
+         * This method is used by Form when scrolling to the input
+         *
+         * @param relativeToNativeComponentRef - reference to an ancestor
+         * @param onSuccess - callback called on success
+         * @param onFail - callback called on failure
+         */
+        measureLayout(relativeToNativeComponentRef, onSuccess, onFail) {
+            if (!root.current) {
+                return;
+            }
+
+            root.current.measureLayout(relativeToNativeComponentRef, onSuccess, onFail);
+        },
+    }));
+
+    const hasError = !!errorText;
+
+    if (isDisabled) {
+        return (
+            <View>
+                {Boolean(label) && (
+                    <Text
+                        style={[styles.textLabelSupporting, styles.mb1]}
+                        numberOfLines={1}
+                    >
+                        {label}
+                    </Text>
+                )}
+                <Text numberOfLines={1}>{value}</Text>
+                {Boolean(hintText) && <Text style={[styles.textLabel, styles.colorMuted, styles.mt2]}>{hintText}</Text>}
+            </View>
+        );
+    }
+
+    return (
+        <>
+            <View
+                ref={root}
+                style={[styles.pickerContainer, isDisabled && styles.inputDisabled, containerStyles, isHighlighted && styles.borderColorFocus, hasError && styles.borderColorDanger]}
+            >
+                {label && (
+                    <Text
+                        pointerEvents="none"
+                        style={[styles.pickerLabel, styles.textLabelSupporting]}
+                    >
+                        {label}
+                    </Text>
+                )}
+                <RNPickerSelect
+                    onValueChange={onValueChange}
+                    // We add a text color to prevent white text on white background dropdown items on Windows
+                    items={items.map((item) => ({...item, color: themeColors.pickerOptionsTextColor}))}
+                    style={size === 'normal' ? styles.picker(isDisabled, backgroundColor) : styles.pickerSmall(backgroundColor)}
+                    useNativeAndroidPickerStyle={false}
+                    placeholder={pickerPlaceholder}
+                    value={value}
+                    // TODO: update the icon logic to escape ts error
+                    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                    // @ts-ignore
+                    Icon={() => icon(size)}
+                    disabled={isDisabled}
+                    fixAndroidTouchableBug
+                    onOpen={enableHighlight}
+                    onClose={disableHighlight}
+                    textInputProps={{
+                        allowFontScaling: false,
+                    }}
+                    pickerProps={{
+                        // TODO: resolve 'ref does not exist in type CustomPickerProps' ts error
+                        // ref: picker,
```

Comment:
@fabioh8010 Maybe you have an idea of what I can do with this `ref does not exist in type CustomPickerProps` error